src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -884,22 +902,41 @@ private boolean updateSchema(Connection connection) throws Exception
 
             if (logger.isInfoEnabled())
             {
-                logger.info("Create scripts executed in "+(System.currentTimeMillis()-start)+" ms");
+                logger.info("Create scripts executed in " + (System.currentTimeMillis() - start) + " ms");
             }
         }
         else
         {
+            long start = System.currentTimeMillis();
+
             // Execute any pre-auto-update scripts
             checkSchemaPatchScripts(connection, preUpdateScriptPatches, true);
             
             // Execute any post-auto-update scripts
             checkSchemaPatchScripts(connection, postUpdateScriptPatches, true);
+
+            if (logger.isInfoEnabled())
+            {
+                logger.info(
+                    "Update case: checkSchemaPatchScripts scripts executed in " + (System.currentTimeMillis() - start) + " ms");

```

Comment:
Can it be rephrased like `Update scripts executed in ...`

Reply:
I can change this message. I run out of inspiration apparently